src/javascript/app/pages/cashier/account_transfer.js

```diff
@@ -2,6 +2,7 @@ const BinaryPjax         = require('../../base/binary_pjax');
 const Client             = require('../../base/client');
 const BinarySocket       = require('../../base/socket');
 const isCryptocurrency   = require('../../common/currency').isCryptocurrency;
+const isStableCoin       = require('../../common/currency').isStableCoin;
 const getMinWithdrawal   = require('../../common/currency').getMinWithdrawal;
```

Comment:
perhaps we should require the entire currency module since we are using three functions from there.